_ext/jellyfin.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: ext
```

Comment:
None of the others have this, does it do anything?

Reply:
It's applied by default by the rules in `_config.yml`, so it is redundant.

Reply:
It was in the example in the readme.  If it's not needed I can pull it out

Reply:
Thanks for pointing that out, I've updated the example. Might as well remove it so Jellyfin is consistent with the others.